### DISCUSSION TITLE
Missing comma

### DISCUSSION THREAD

**Discussion by LokiNaBoki** [nvim-lualine/lualine.nvim#396] [Configuration]
Hi, I just want say that you are missing a coma in ["default config" in Read.me](<https://github.com/nvim-lualine/lualine.nvim/blob/483af24670ed1e3951d20ec3ce5fc6305bd91daf/README.md?plain=1#L125>), line: `disabled_filetypes = {}` ends with just new line even though there is one more line after it in the array. I believe It's not big enough for issue.

**Comment by shadmansaleh** [comment]
done https://github.com/nvim-lualine/lualine.nvim/commit/ef063f72b48caf9c31dec670ddc489f05bad0949

> I believe It's not big enough for issue

A pr would have been nice though 😄
